WindSpigot-Server/src/main/java/org/bukkit/craftbukkit/CraftServer.java

```diff
@@ -826,7 +826,6 @@ public void reload() {
 		org.spigotmc.SpigotConfig.init((File) console.options.valueOf("spigot-settings")); // Spigot
 		org.github.paperspigot.PaperSpigotConfig.init((File) console.options.valueOf("paper-settings")); // PaperSpigot
 		net.techcable.tacospigot.TacoSpigotConfig.init((File) console.options.valueOf("taco-settings")); // TacoSpigot
-		NachoConfig.init((File) console.options.valueOf("nacho-settings")); // NachoSpigot
```

Comment:
Why is the NachoSpigot config loaded on startup but not on reloads?

Reply:
Currently the nacho config is only loaded for migration to wind

Reply:
Well yeah, that's the final goal, but that has not been finished yet.